src/lib/strategy-service.ts

```
@@ -829,31 +920,51 @@ class StrategyService {
 
   async deactivateStrategy(id: string): Promise<Strategy> {
     try {
-      // Try multiple approaches to update the database
+      // Initialize API client if not already initialized
+      await apiClient.initialize();
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)